vendor/engines/secure_rooms/db/migrate/20170525143813_add_token_to_secure_room.rb

```diff
@@ -0,0 +1,14 @@
+class AddTokenToSecureRoom < ActiveRecord::Migration
+
+  class Product < ActiveRecord::Base
+  end
+
+  def change
+    add_column :products, :dashboard_token, :string
+    add_index :products, :dashboard_token
+
+    Product.reset_column_information
+    Product.where(type: :SecureRoom).find_each { |room| room.update_column(:dashboard_token, SecureRandom.uuid) }
```

Comment:
And will this `db:rollback` properly? I always forget when you run some extra code.